.github/workflows/ci.yaml

```diff
@@ -50,6 +50,13 @@ jobs:
           sudo pip install podman-compose
           podman --version
 
+      # https://bugs.launchpad.net/ubuntu/+source/libpod/+bug/2024394
+      - name: fix podman
+        if: matrix.TEST == 'podman' && matrix.os == 'ubuntu-latest'
```

Comment:
Instead of testing for ubuntu-latest (which is insufficient), check the current podman version in shell logic.:
https://github.com/pulp/pulp-oci-images/blob/latest/.github/workflows/pulp_images.yml#L193

Specify: `shell: bash`

This is important so as to not break the CI in a matter of weeks.